### ISSUE TITLE
Add README to explain how to reproduce examples

### ISSUE THREAD

**Issue by wh81752** [Mogztter/asciidoctor-web-pdf#457]
I tried to reproduce examples, especially Lea Rumiz's resume. It is not intuitive how to redo, eventually I ended up in:

```
$> asciidoctor-web-pdf resume.adoc \
  -a stylesheet="style.css"\
  --template-require ./template.js
```

Perhaps a simple README.md per example would help to clarify how to go ahead.

**Comment by Mogztter** [comment]
> Perhaps a simple README.md per example would help to clarify how to go ahead.

Yes that's a good idea 👍🏻

**Comment by Mogztter** [comment]
fixed in  36b4d59
